test/sharness/t0250-files-api.sh

```diff
@@ -629,6 +629,20 @@ test_files_api() {
   test_expect_success "repo gc $EXTRA" '
     ipfs repo gc
   '
+
+  # test rm
```

Comment:
I'd like to add a test with an actual corrupted node, which could just be an empty proto-node, but I'm not sure how could we add that to MFS, `ipfs files cp` does check the type:
```
ipfs object new
# QmdfTbBqBPQ7VNxZEYEj14VmRuZBkqFbiwReogJgS1zR1n
ipfs files cp /ipfs/QmdfTbBqBPQ7VNxZEYEj14VmRuZBkqFbiwReogJgS1zR1n /corrupted
# Error: cp: cannot flush the created file /corrupted: proto: required field "Type" not set
```

Reply:
Oh, turn off flush:

```
ipfs files -f=0 cp /ipfs/QmdfTbBqBPQ7VNxZEYEj14VmRuZBkqFbiwReogJgS1zR1n /corrupted
ipfs files rm /corrupted
Error: proto: required field "Type" not set
```

Reply:
OT: What's the difference of adding a file in MFS without flushing? I still don't know how that works (and I'm supposedly the one in charge of reviewing the MFS API :grimacing:)

Reply:
Thinking about this some more, after adding the checks to `PutNode` this won't work anyways (even with flush turned off), so scratch that, I'm fine with the existing tests.